### PR TITLE
18 upload audio presentation

### PR DESCRIPTION
**Pull request by jvcanavarro**

### CODE REVIEW

File: mobile/lib/features/text_management/presentation/widgets/text_view_popup_menu_button.dart [hunk 1]
```diff
@@ -34,14 +37,21 @@ class TextViewPopupMenuButton extends StatelessWidget {
...
-              value: BlocProvider.of<TextBloc>(context),
-              child: AudioPage(
-                myTextTest: text,
+            builder: (_) => BlocProvider(
+              create: (_) => GetIt.instance<AudioBloc>(
+                param1: text,
+                // test student entity
+                param2: Student(
+                  classroomId: 0,
+                  firstName: 'cu',
```
Comment (FelipOliveira) [inline mobile/lib/features/text_management/presentation/widgets/text_view_popup_menu_button.dart:49]: what is it? why did you put this?

**Review by FelipOliveira — COMMENTED** [review]
some changes are needed in 'upload audio'

File: mobile/lib/features/text_management/presentation/widgets/text_view_popup_menu_button.dart (hunk 1 above)
Comment (jvcanavarro) [reply mobile/lib/features/text_management/presentation/widgets/text_view_popup_menu_button.dart:49]: xD

**Comment by jvcanavarro** [comment]
Closes until further tests with another devices

**Pull request closed without merging**